src/main/webapp/app/modules/upload/upload-page.tsx

```diff
@@ -109,9 +109,21 @@ export class UploadPage extends React.Component<IUploadPageProp, IUploadState> {
 
   fileValidateTypeDetectType = (source, type) =>
     new Promise((resolve, reject) => {
+      type = this.handleUndefinedFileType(source, type);
       resolve(type);
     });
 
+  handleUndefinedFileType = (source, type) => {
+    if (type === '') {
+      const re = /(?:\.([^.]+))?$/;
+      const fileExtension = re.exec(source.name)[1];
+      if (fileExtension === 'csv') {
+        type = 'text/csv';
```

Comment:
Can only CSV be undefined?

Reply:
As far as I know in our case only CSV file may cause problems.